components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -504,9 +563,13 @@ public RuleTemplate getRuleTemplate(String templateGroupUUID, String ruleTemplat
     }
 
 
-    private Artifact buildSiddhiAppFromScratch(Map<String, Artifact> derivedTemplates, BusinessRuleFromScratch businessRuleFromScratch)
+    private Artifact buildSiddhiAppFromScratch(Map<String, Artifact> derivedTemplates,
+                                               BusinessRuleFromScratch businessRuleFromScratch)
             throws
             TemplateManagerException {
+        ClassLoader classLoader = TemplateManagerService.class.getClassLoader();
+        String SIDDHI_APP_TEMPLATE = classLoader.getResource("siddhi-app-template.json").getFile();
```

Comment:
Please use constant for file name "siddhi-app-template.json"

Reply:
Fixed from df3bc5a752f0dfcd03393c5f62409814722e720b